### ISSUE TITLE
Asset Kinds

### ISSUE THREAD

**Issue by FreneticScribbler** [nottinghamtec/PyRIGS#452]
At present, the assets database doesn't know when a particular set of assets are all the same, say all the telehoists, or all the DL4s.

There is definite utility in putting this information in somehow. Some things, like weight, required tests etc, can and should be stored at this slightly higher/more abstract level. It would also represent an easy way to query 'all DL4s' or similar, as at present the only way to do this is text search. Bleh. 

Essentially the proposal is to expand Cable Types to all assets but I'm unsure of how exactly to implement it. Make and 'model'??

Basically just dropping this here as a reminder to myself to have a discussion about it at some point...
